openstack/networking/v2/extensions/security/rules/requests.go

```diff
@@ -145,6 +145,11 @@ func Create(ctx context.Context, c *gophercloud.ServiceClient, opts CreateOptsBu
 		r.Err = err
 		return
 	}
+	if m, mOk := b["security_group_rule"].(map[string]any); mOk {
```

Comment:
it'd be better to add this into the `ToSecRuleCreateMap`

Reply:
done